### PR TITLE
MarkovModel.py: Updated check_model and get_cardinality method.

### PR DESCRIPTION
**Pull request by lohani2280**
Fixes https://github.com/pgmpy/pgmpy/issues/816

### CODE REVIEW

File: pgmpy/models/MarkovModel.py [hunk 1]
```diff
@@ -226,6 +218,8 @@ def check_model(self):
                 if cardinalities[variable] != cardinality:
                     raise ValueError(
                         'Cardinality of variable {var} not matching among factors'.format(var=variable))
+                if len(self.nodes()) != len(cardinalities):
```
Comment (ankurankan) [inline pgmpy/models/MarkovModel.py:221]: @lohani2280 The PR looks good. Could you just add a new test for these two new lines ?

**Review by ankurankan — CHANGES_REQUESTED** [review]

File: pgmpy/models/MarkovModel.py (hunk 1 above)
Comment (lohani2280) [reply pgmpy/models/MarkovModel.py:221]: @ankurankan Yes, i'll do it very soon :)

**Comment by ankurankan** [comment]
@lohani2280 What happened with this PR ?

**Comment by lohani2280** [comment]
I did some blunders in my local branch and there were lots of merge conflicts ,therefore, i closed this PR and send another PR https://github.com/pgmpy/pgmpy/pull/892.

**Pull request closed without merging**